plugins/woocommerce/client/admin/client/wp-admin-scripts/settings-embed/settings-ui.scss

```
@@ -65,12 +73,52 @@ body.woocommerce_page_wc-settings.woocommerce-settings-ui-page {
 	}
 
 	.wc-settings-ui-shell__header-actions {
+		display: flex;
 		flex: 0 0 auto;
+		gap: var(--wpds-dimension-gap-sm, 8px);
 		margin-inline-start: auto;
 	}
 
 	.wc-settings-ui-shell__subtitle {
 		color: var(--wpds-color-foreground-content-neutral-weak);
```

Stylelint (17.14.0)

[error] The following CSS variables are not valid Design System tokens: '--wpds-color-foreground-content-neutral-weak' (plugin-wpds/no-unknown-ds-tokens)

(plugin-wpds/no-unknown-ds-tokens)